### ISSUE TITLE
Clean database on Android

### ISSUE THREAD

**Issue by cbenhagen** [hnvn/flutter_downloader#2]
As far as I can see old tasks pile up and will not get cleaned from the database. Or do I miss something?

**Comment by hnvn** [comment]
on Android, every task information is saved to DB and I don't think that we need to clean it.

**Comment by cbenhagen** [comment]
Sorry I don't see the point why we should keep old and even failed tasks around forever. Can you explain in what circumstances that could make sense? I mean even if there is a use case for it. There should at least be a way to clean it.

**Comment by hnvn** [comment]
I agree with you that tasks already finished (with or without error) can be free to clean up. But I wonder about the condition to clean up tasks and when it should do. I think we just keep it simple and consider it like logs instead of paying attention to decide a task can be cleaned or not. Btw, task data is lightweight, it wouldn't cause any problem with your device space

**Comment by cbenhagen** [comment]
It does not matter how lightweight logs are. At some point they will fill your device when they are not rotated. I have seen servers run out of space because of logs. After some testing my `download_tasks.db` is already more than 5MB big. @hnvn please consider to reopen this issue.

**Comment by codmajik** [comment]
Hi, there is also another part that needs considering, which is that loadTask returns all tasks, not providing a way to clean completed tasks means that eventually you app would be iterating over 5MB worth of task info just to find the task. @hnvn please consider a function to clear old downloads even if it is just a way to flag it so it doesn't show up in loadTask would be fine.

**Comment by sriharsha-y** [comment]
@hnvn Still there is no way to cleanup db ?

**Comment by Samurai016** [comment]
I found this method, it's working for me
``` Dart
var query = "DELETE FROM task";
FlutterDownloader.loadTasksWithRawQuery(query: query);
```
